frontend/components/App.tsx

```diff
@@ -0,0 +1,52 @@
+import React, {useState, useEffect} from 'react';
+import { BrowserRouter as Router, Route, Switch, Redirect} from 'react-router-dom';
+import * as ROUTES from '../constants/routes';
+
+import LandingPage from './LandingPage/LandingPage';
+import Main from './Main/Main';
+
+import firebase from './Firebase';
+import LoadingContext from "./LandingPage/context";
+import AuthUserContext from "./Session/context";
+
+const App = () => {
+  const [authUser, setAuthUser] = useState(null);
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => firebase.auth.onAuthStateChanged(
```

Comment:
Wait we're still using Firebase? I thought that we were switching to Platform authentication and we only needed Firebase for authentication.

Just want to make sure this is eventually removed.